PowerShell-Win.sln

```diff
@@ -3,31 +3,33 @@ Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio 15
```

Comment:
Can we have a comment here pointing to https://github.com/dotnet/project-system/blob/master/docs/opening-with-new-project-system.md#project-type-guids ?